src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1248,14 +1217,15 @@ private void CalculateOrganelleEffectivenessInPatch(Patch patch = null)
         patch ??= CurrentPatch;
 
         gui.UpdateEnergyBalance(
-            ProcessSystem.ComputeEnergyBalance(organelles.Select(i => i.Definition), patch.Biome, membrane));
+            ProcessSystem.ComputeEnergyBalance(organelles, patch.Biome, membrane));
     }
 
     private void CalculateCompoundBalanceInPatch(List<OrganelleTemplate> organelles, Patch patch = null)
     {
         patch ??= CurrentPatch;
 
-        var result = ProcessSystem.ComputeCompoundBalance(organelles.Select(i => i.Definition), patch.Biome);
+        var result = ProcessSystem
+            .ComputeCompoundBalance(organelles.Select(organelle => organelle.Definition), patch.Biome);
```

Comment:
There are two calls to ProcessSystem.ComputeXBalance functions here, but one of them takes an `OrganelleTemplate` list, and the other the list of their definitions. I think some consistent choice should be made here.